tests/main/apparmor-prompting-integration-tests/read_single_allow.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/sh
+
+# A simple allow once test of a read prompt.
+
+TEST_DIR="$1"
+TIMEOUT="$2"
+if [ -z "$TIMEOUT" ] ; then
+	TIMEOUT=10
+fi
```

Comment:
are you expecting people to invoke the scripts directly, so that's why this is here?

Reply:
If there's no second argument, then TIMEOUT would be `""` which would break the test, so I figured it would be safer to ensure TIMEOUT is set. And yes, I've found it useful to manually invoke these test case scripts when testing in a lxd VM, for example.